example/4ebp2_example/README.rst

```diff
@@ -0,0 +1,49 @@
+Building Folded Regions - Beads on a String Model (FLDR/S)
+==========================================================
+
+.. start-description
+
+The following example will walk you through building a folded domain within the
+internal coordinate space upon which IDPConformerGenerator natively operates within.
+
+For this exercise, please enter the :code:`example/4ebp2_example` folder where you
+will find the FASTA sequence: ``4ebp2.fasta``, and a tarball containing snapshots of the
+folded domain extracted from PDB ID 2MX4: ``2MX4_4ebp2.tar``.
+
+Steps from now will assume you're in the working director of ``example/4ebp2_example``
+and have already compiled your preferred reusable IDPConformerGenerator database. For
+instructions on the database, please visit the previous exercise "A Real Case Scenario".
+
+Create your folded domain database using the following commands::
+
+    idpconfgen sscalc 2MX4_4ebp2.tar -n
```

Comment:
Remember here to add the `-cmd mkdssp` at leat between brackets.

Reply:
Aha yes, I need to update the documentation also. Some users were confused with `<DSSP CMD>`

Reply:
Yes, very user I've interacted with didn't know <DSSP CMD> was optional, I think we should do `-cmd mkdssp` to be safe as DSSP v4.X input is different than our compatible v3.X